hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -121,7 +121,7 @@ public boolean isTimeToExport(Instant now) {
 		}
 		periodBegin = now;
 		if(log.isDebugEnabled()) {
-			log.debug(String.format("Now %s is NOT time to export.", now.toString()));
+			log.debug("Now " + now.toString() + " is NOT time to export.");
```

Comment:
I don't think the log messages need to be changed, I think it is OK to remain as they were if you want.

Reply:
Why would use either approaches when log4j has variable replacement using `{}`? And it's redundant to call toString() on now.

Reply:
Made the changes, thanks!